experimental/dds/tree2/src/test/snapshots/testTrees.ts

```diff
@@ -32,7 +33,10 @@ import {
 import { AllowedUpdateType, FieldKey, JsonableTree, UpPath, rootFieldKey } from "../../core";
 import { leaf } from "../../domains";
 
-const factory = new SharedTreeFactory({ jsonValidator: typeboxValidator });
+const factory = new SharedTreeFactory({
+	jsonValidator: typeboxValidator,
+	summaryEncodeType: TreeCompressionStratagy.Compressed,
```

Comment:
As we store textual diffs for these, if they are going to explicitly an option, I think uncompressed would make more sense

Reply:
Updated to `TreeCompressionStrategy.Uncompressed`